packages/warehouses/src/warehouseClients/PostgresWarehouseClient.ts

```diff
@@ -314,6 +314,13 @@ export class PostgresClient<
         if (databases.size <= 0 || schemas.size <= 0 || tables.size <= 0) {
             return {};
         }
+
+        const { rows: supportsMatViewsRows } = await this.runQuery(
+            `SELECT current_setting('server_version_num')::integer >= 120000 as supports_matviews`,
```

Comment:
so this is >= v12?

Reply:
yep yep, `pg_matviews` is only available on and after 12: https://www.postgresql.org/docs/current/view-pg-matviews.html